orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/admin/web/QueueAdminController.kt

```diff
@@ -51,6 +55,17 @@ class QueueAdminController(
       )
     )
 
```

Comment:
Would also like to see a `killZombie/{executionId}` so we can kill a specific execution (which i think is more common of a scenario [and safer])

Reply:
Added.